src/cmd/cmd.zig

```diff
@@ -47,8 +47,14 @@ const config = @import("config.zig");
 const ACCOUNT_INDEX_BINS = @import("../accountsdb/db.zig").ACCOUNT_INDEX_BINS;
 const socket_tag = @import("../gossip/data.zig").socket_tag;
 
+// TODO: use better allocator, unless GPA becomes more performant.
+
 var gpa = std.heap.GeneralPurposeAllocator(.{}){};
 const gpa_allocator = gpa.allocator();
+
+var table_gpa: std.heap.GeneralPurposeAllocator(.{}) = .{};
+const table_gpa_allocator = table_gpa.allocator();
```

Comment:
can we change `table` to `value` so its `value_allocator` since it allocates gossip values